schema/src/main/java/org/hl7/v3/ObjectFactory.java

```diff
@@ -3522,7 +3522,7 @@ public RCMRIN030000UK06Subject createRCMRIN030000UK06Subject() {
     }
 
     @XmlElementDecl(namespace = "urn:hl7-org:v3", name = "addr")
-    public JAXBElement<AD> createPatient(AD address) {
+    public JAXBElement<AD> createAddress(AD address) {
```

Comment:
What is the reason for this change? (I mean it looks correct to be changed? But no documentation change? And we're not using it?) All the questions :D

Reply:
Was named wrong in my last PR, it's used in testing the AddressUtil for extracting the address test xml to java object